src/zenml/integrations/aws/secret_schemas/aws_secret_schema.py

```diff
@@ -27,3 +27,11 @@ class AWSSecretSchema(BaseSecretSchema):
     aws_access_key_id: str
     aws_secret_access_key: str
     aws_session_token: Optional[str]
+
+    class Config:
```

Comment:
This exact `Config` was added to all secret schemas except the newly created `MYSQLSecretSchema`. Is there any reason for that, and if not can we add this config once to the `BaseSecretSchema` instead?

Reply:
Great idea

Reply:
Done